src/components/_homepage/testimonials.module.scss

```diff
@@ -1,211 +1,186 @@
 @import '../../base/module';
 
-.flex {
-  display: block;
+.wrapper {
+  display: flex;
+  flex-direction: column-reverse;
+  justify-content: space-between;
+  align-items: center;
+  gap: 40px;
 
-  @media (min-width: $tablet) {
-    display: flex;
+  @media (min-width: 800px) {
+    flex-direction: row-reverse;
+    padding: 0 15px;
   }
 }
+.background {
+  background-color: $color-darkgreen;
 
```

Comment:
The container here needs position relative for the palm to work properly but when I do it it and have it overhang on the bottom it forces a scroll bar and if I get rid of the scrollbar then it forces it to cutoff, can't get it to just overhang down.